workshops/image_editor/README.md

```diff
@@ -0,0 +1,451 @@
+        <form id="imgurl">
+              <input type="text" placeholder="Image URL" id="onlineurl">
+              <button type="submit">Change Image</button>
+          </form>
+      </div>
+  </div>
+  
+  <!--Section #2-->
+  <div class="row">
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4 text-center">
+      <img src="https://assets.hackclub.com/flag-orpheus-top.svg" id="targetimage" alt="">
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+
+  <!--Section #3-->
+  <div class="row">
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4 p-2 text-center" id="section_three">
+      <h3>Image Filters</h3>
+            <form id="slider-form">
+            <p>
+              <label for="gs">Grayscale</label>. 
+              <input id="gs" name="gs" type="range" min="" max="100" value="0" class="slider">
+            </p>
+                                
+            <p>
+              <label for="blur">Blur</label>
+              <input id="blur" name="blur" type="range" min="0" max="" value="0" class="slider">
+            </p>
+
+            <p>
+              <label for="hue-rotate">Hue-Rotate</label>
+              <input id="hue-rotate" name="hue-rotate" type="range" min="0" max="" value="0" class="slider">
+            </p>
+
+            <p>
+              <label for="sepia">Sepia</label>
+              <input id="sepia" name="sepia" type="range" min="0" max="" value="0" class="slider">
+            </p>
+            
+            <button type="reset" class="btn btn-primary" >Reset</button>
+        </form>
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+</div>
+```
+</details>
+
+![Section 3 Result](https://cloud-jwbtp72pu.vercel.app/0screen_shot_2020-11-11_at_12.33.02_pm.png)
+
+# CSS
+The Boostrap stylesheet we added at the beginning of this workshop gives us most of the styling we need. But let's do a little bit of our own customizing.
+
+Open the `style.css` file on the sidebar on the left. Then, add:
+
+```css
+img {
+  width: 400px;
+}
+
+#section_three {
+  border: 2px solid black; 
+}
+```
+Let's break this down:
+- First, we set `400px` as the default width for every image we edit because some images can be too big and otherwise might not fit on the website.
+- Then, we add a border to the section 3 to make it look a little bit better.
+
+Here's the result:
+![CSS Result](https://cloud-i0vj1gzw6.vercel.app/0image.png)
+
+Repl.it should have automatically linked the `style.css` file for you, but if you don't see your changes reflect when you click the green "Run" button at the top, try adding this line to the `<head>` in your `index.html` file:
+
+```html
+<link rel="stylesheet" href="style.css">
+```
+
+# JavaScript
+This is my favorite part of the workshop, because this part makes the website actually work.
+
+Navigate to the `script.js` file in your sidebar. Then, create these 3 variables:
+
+```javascript
+let myform = document.getElementById('imgurl');
```

Comment:
One thing I've noticed throughout the workshop is that all of your variables are all lowercase, but generally in JavaScript they're in camel case (so like myForm instead of myform). I know this is nitpicky but can you go through and change all instances of all-lowercase variables and functions to make them camel case?